include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -598,15 +598,15 @@ __pattern_transform_scan_single_group(_ExecutionPolicy&& __exec, _InRng&& __in_r
             if (__is_full_group)
                 return __parallel_transform_scan_single_group_submitter<
                     _Inclusive::value, __num_elems_per_item, __wg_size,
-                    /* _IsFullGroup= */ true>::__launch_static_bounds_scan(__exec, __in_rng.all_view(),
-                                                                           __out_rng.all_view(), __n, __init,
-                                                                           __binary_op, __unary_op);
+                    /* _IsFullGroup= */ true>::__launch_static_bounds_scan(__exec, std::forward<_InRng>(__in_rng),
+                                                                           std::forward<_OutRng>(__out_rng), __n,
+                                                                           __init, __binary_op, __unary_op);
             else
                 return __parallel_transform_scan_single_group_submitter<
                     _Inclusive::value, __num_elems_per_item, __wg_size,
-                    /* _IsFullGroup= */ false>::__launch_static_bounds_scan(__exec, __in_rng.all_view(),
-                                                                            __out_rng.all_view(), __n, __init,
-                                                                            __binary_op, __unary_op);
+                    /* _IsFullGroup= */ false>::__launch_static_bounds_scan(__exec, std::forward<_InRng>(__in_rng),
+                                                                            std::forward<_OutRng>(__out_rng), __n,
```

Comment:
`std` -> `::std` for these and later on as well.

Reply:
Fixed.